src/main/java/com/michelin/ns4kafka/controllers/ConnectClusterController.java

```diff
@@ -139,4 +147,61 @@ public HttpResponse<Void> delete(String namespace, String connectCluster, @Query
         connectClusterService.delete(connectClusterToDelete);
         return HttpResponse.noContent();
     }
+
+    /**
+     * Gets the list of available Kafka Connect vaults for a namespace
+     *
+     * @return The list of the available Kafka Connect vaults.
+     */
+    @Get("/vaults")
```

Comment:
`@Get("/_/vaults")` is required (see `imports` endpoint) to match role bindings criteria